recipes/perl-uri/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "perl-uri" %}
+{% set version = "5.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-5.17.tar.gz
+  sha256: 5f7e42b769cb27499113cfae4b786c37d49e7c7d32dbb469602cd808308568f8
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - make
+  host:
+    - perl 5.32.1
```

Comment:
Please never add explicit Perl version requirements. The Perl version is defined by the global version pinning of CondaForge. The run export should ensure that you get the right Perl version at runtime.